recipes/cbflib/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "CBFlib" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/yayahjb/cbflib/archive/refs/tags/CBFlib-{{ version }}.tar.gz
+    folder: cbflib
+    sha256: e4acf8f7b5cc4e788215983ff428c2f8aa1fdbbfc6d163ace7999dcd2a10b19f
+    patches:
```

Comment:
What's the source of these patches? Can you add some attribution to them (e.g. a `git format-patch` export)?

Reply:
These patches essentially update the CMake support to build on Windows and make the test suite pass. I put them together with help from for instance https://github.com/dials/pycbf (credited in `09-pycbf-dials.patch`). I will submit them upstream, but given the volume they will have be split into several pull requests and it may be a while before all that's merged.